corehq/privileges.py

```diff
@@ -85,6 +85,8 @@
 
 LITE_RELEASE_MANAGEMENT = 'lite_release_management'
 
+LOADTEST_USERS = 'loadtest_users'
```

Comment:
hey @kaapstorm! is it possible to PR this migration of adding a new privilege separately so that it can quickly get merged and unblock any merge conflicts happening with migrations on staging? I think on SaaS we generally have a policy of putting migrations in PRs separate from code changes to avoid issues like that and also ensure that the code changes can be easily reverted if needed. Thank you! ^_^

Reply:
That is an excellent idea! Thank you, will do.